defs/ui_hooks_oas.yaml

```diff
@@ -126,7 +126,7 @@ components:
       schema:
         type: string
       description: >-
-        The URL of the URL attachment on the task (i.e. Jira issue, Github pull request)
+        The URL of the URL attachment on the task (i.e. Jira issue, GitHub pull request)
```

Comment:
This change is in a .yaml file that requires the rest of the toolchain to deploy. The change here can stay, but it might not be fixed in the docs until we do a larger deploy.